index.js

```diff
@@ -1,10 +1,41 @@
 /**
  * The function below has no parameters. Instead, access the arguments from the command line.
- * The first argument passed after the filename should be either "plus" or "minus", which represents the kind of calculation that will be done. Every argument afterwards should be a number.
+ * The first argument passed after the filename should be either "plus" or "minus", which represents 
+ * the kind of calculation that will be done. Every argument afterwards should be a number.
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
- * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
+ * @returns {number} The result of either adding all numbers or subtracting all numbers,
+ * depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    if (process.argv.length == 2) {
+        return "No operation provided..."
+    }
+
+    if (process.argv.length == 3) {
+        return "No numbers provided..."
+    }
+
+    if (process.argv.length >= 4) {
+        let total = 0;
+
+        if(process.argv[2] == "plus"){
```

Comment:
The code in this else if block and the one on line 25 are very similar with few differences. How can you refactor this code to be less repetitive??

Reply:
Done. Does this work better?